dts/bindings/kscan/ite,it8xxx2-kscan.yaml

```diff
@@ -14,6 +14,12 @@ properties:
     interrupts:
       required: true
 
+    wucctrl-0:
+      type: phandles
+      description: |
+        Pin configurations for the state. Content is specific to the
+        selected pin controller interface implementation.
```

Comment:
This looks a bit odd. If it's pinctrl-related, why isn't this using the standard `pinctrl-0` property?

CC @gmarull

Reply:
Wakeup controller interface is used to set that when the interrupt is triggered in EC low power mode, it can wakeup EC or not. Via this interface, we set the wakeup trigger edge, enable and disable for the specific pin which may be gpio pins or alternate pins. Wakeup controller interface is specific for ITE and the register block is different from pinmux and gpio. I update the description.